api/walletApi-V0.yml

```diff
@@ -163,22 +169,26 @@ paths:
           description: Unprocessable Entity - syntax is correct, but the server was unable to process the contained instructions.
           schema:
             $ref: "#/definitions/Error"
+        "423":
+          description: Locked - The resource that is being accessed is locked
```

Comment:
Why lock ? Where can I see the fnctional rules around thsi concept ?

Reply:
maybe @SlnPons you can help me here ?

Reply:
Sure, we have introduce the notion of "locked account" to prevent people from accessing and performing actions on other's accounts if they stay open on a browser windows by mistake. 
This functionality works like this: 
* As a user, if I do NOT perform any action on my account for more than 10min I see my account screen locked.
* As a user, if I want to unlock my screen to perform actions I need to enter my account password using password prompt (wails). 
Here are the design in staging status (not validated yet): https://www.figma.com/proto/LSaOibVDFb1reF6FePVn5l/Massastation?node-id=515-6325&scaling=contain&page-id=324%3A5833&starting-point-node-id=476%3A4352